macros/utils/common_test_configs.sql

```diff
@@ -0,0 +1,393 @@
+{% macro get_common_test_config_by_flattened_test(flattened_test) %}
+  {% set test_name = flattened_test['short_name'] %}
+  {% set test_namespace = flattened_test['test_namespace'] %}
+  {% do return(elementary.get_common_test_config_by_namespace_and_name(test_namespace, test_name)) %}
+{% endmacro %}
+
+{% macro get_common_test_config_by_namespace_and_name(test_namespace, test_name) %}
```

Comment:
I think we can remove the `__by_namespace_and_name` from the macro's name.
Typically the parameters received by a macro shouldn't be repeated in its name.

Reply:
the issue is i have both this and `get_common_test_config_by_flattened_test`, and if i remove it from both of them then i have two functions with the same name (which i dont believe is possible in jinja)